stream-chat-android-offline/src/main/java/io/getstream/chat/android/offline/ChatDomainImpl.kt

```diff
@@ -691,8 +695,11 @@ internal class ChatDomainImpl internal constructor(
                 val channelController = this.channel(c)
                 channelController.watch()
             }
-            // 4. recover events (async)
-            replayEventsForChannels(cids)
+        }
+        // 4. recover missing events
+        val activeChannelCids = getActiveChannelCids()
+        if (activeChannelCids.isNotEmpty()) {
```

Comment:
Why do you need this check? Seems the method works fine with the empty collection:
```kotlin
 replayEventsForChannels(getActiveChannelCids())
```